### ISSUE TITLE
Failed to build 

### ISSUE THREAD

**Issue by antfu** [vuejs/vitepress#47]
**Describe the bug**
On current `master`, `yarn build` failed due to typing error.

```
src/client/app/utils.ts:20:32 - error TS2339: Property 'env' does not exist on type 'ImportMeta'.

20       const base = import.meta.env.BASE_URL
                                  ~~~

Found 8 errors.
```

**To Reproduce**
- Clone, `yarn` and then `yarn build`

**Additional context**
Fix PR made: https://github.com/vuejs/vitepress/pull/44

**Comment by antfu** [comment]
Temp workaround: 

Copy https://github.com/vitejs/vite/blob/master/importMeta.d.ts and put it under `src/`

**Comment by yyx990803** [comment]
Resolved in 583f02e
